MiniApp/Classes/Display/MiniAppExternalWebViewController.swift

```diff
@@ -53,6 +54,11 @@ class MiniAppExternalWebViewController: UIViewController {
             self.webView.load(URLRequest(url: url))
         }
     }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(true)
+        self.miniAppExternalUrlLoader = nil
```

Comment:
this is working and it's cool but can't be a permanent fix, because if you loose the reference to `miniAppExternalUrlLoader`, viewdiddisappear does not guarantee your controller needs to be destroyed, the controller may be brought back to the user into a navigation for example, and then the MiniApp will never have the feedback.

Reply:
That's right. This wasn't a permanent fix. We need to realign the way External Webview is loaded. But as for this fix loosing `miniAppExternalUrlLoader` reference at `viewDidDisappear` will make the `MiniAppExternalWebViewController` release immediately. Not sure why `miniAppExternalUrlLoader` holding a strong reference, which I need to investigate further